### PR TITLE
Update "Getting started" section.

### PR DESCRIPTION
**Pull request by lalitb**
Update the `Getting Started` section to enable users with a Microsoft alias (including employees and contractors) to reach out to internal community channels and join community meeting invites. The internal links will be eventually updated (as a separate PR) in the private `cpp_client_telemetry_modules` repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,30 @@ data pipeline.
 
 ## Getting Started
 
+### New customer?
+
+ - If you do not have a Microsoft alias, this SDK is not intended for your use. However, you might find the source code useful for understanding how Microsoft collects telemetry.
+
+ - Users with [Microsoft Organization GH account](https://opensource.microsoft.com/), please refer to the link [here](https://github.com/microsoft/cpp_client_telemetry_modules/blob/master/CONTRIBUTING.md) for internal community discussion channels.
```
Comment (lalitb) [inline README.md:25]: The link mentioned here would be updated as separate PR to the private repo.

**Review by ThomsonTan — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -18,8 +18,30 @@ data pipeline.
 
 ## Getting Started
 
+### New customer?
+
+ - If you do not have a Microsoft alias, this SDK is not intended for your use. However, you might find the source code useful for understanding how Microsoft collects telemetry.
```
Comment (reyang) [inline README.md:23]: It might be unclear what does "Microsoft alias" mean.

Comment (lalitb) [reply README.md:23]: Yeah I am not sure how to frame it better - intention is to use include employees and vendors/contractors.

Comment (lalitb) [reply README.md]: ok, shamelessly copied from - https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Exporter.OneCollector#onecollector-exporter-for-opentelemetry-net. Thanks @cijothomas. Let me know if this is fine.

Comment (cijothomas) [reply README.md]: All credits to @CodeBlanch who made the original wording!

**Review by reyang — APPROVED** [review]

**Comment by lalitb** [comment]
Only document update, skipping the tests for merge.
